jobs/build-cosa.Jenkinsfile

```diff
@@ -44,6 +44,22 @@ properties([
     durabilityHint('PERFORMANCE_OPTIMIZED')
 ])
 
+node {
+    change = checkout(
+        changelog: false,
```

Comment:
Minor: I'd just keep this. I think this makes it so that you can see what new commits are in a build from the job status page, which is handy.